components/Breadcrumb/src/story.js

```diff
@@ -1,3 +1,3 @@
-import Tooltips from './Tooltips.js';
+import DenhaagBreadcrumbs from './DenhaagBreadcrumbs';
 
-window.addEventListener('DOMContentLoaded', () => Tooltips());
```

Comment:
@paulAcato weten we zeker dat de verwijderde ToolTips functionaliteit niet nog ergens anders wordt gebruikt?

Reply:
@bleddynacato , de ToolTip JS is op dit punt (binnen de breadcrumb story) gemaakt en dermate strict op breadcrumbs HTML gemaakt dat ik dit heb samengevoegd met de BreadCrumb class vanuit het Thema + heb getweaked ;-) 

Kortom, de Tooltips blijven werken

Reply:
ok!